onyo/shared_arguments.py

```diff
@@ -4,8 +4,20 @@
     action='append',
     type=str,
     help=r"""
-        Use the given **MESSAGE** as the commit message (rather than the default).
+        Append the given **MESSAGE** to the commit message.
```

Comment:
I know this is something that was probably written by myself, but while reading it I thought `Append **MESSAGE** to commit message.` is better, because it is shorter and sounds more like the help below.

Reply:
I agree on the more terse language. I've been noticing other small programmer-isms in the help text, and it is good to remove them.